src/signing/APNSDefaultTokenFactory.php

```diff
@@ -4,13 +4,6 @@
 use \Firebase\JWT\JWT;
 
 class APNSDefaultTokenFactory implements APNSTokenFactory {
-
```

Comment:
We don't actually reference `$credentials` anywhere in the code – this constructor was left over from a previous implementation.